app/controllers/tag_layout_templates_controller.rb

```diff
@@ -4,14 +4,15 @@
 # This class handles creating and viewing Tag Layout Templates, which describe how
 # the Tags in a Tag Group are to be laid out in their labware container. For example
 # that they should be laid out column by column across a plate.
-# Initially the direction algorithm choices have been restricted to a small subset,
-# and the walking direction is fixed to wells of plate.
+# NB. Not all combinations will be valid.
 class TagLayoutTemplatesController < ApplicationController
   DIRECTIONS = {
     'InColumns (A1,B1,C1...)': 'TagLayout::InColumns',
     'InRows (A1,A2,A3...)': 'TagLayout::InRows',
     'InInverseColumns (H12,G12,F12...)': 'TagLayout::InInverseColumns',
-    'InInverseRows (H12,H11,H10...)': 'TagLayout::InInverseRows'
+    'InInverseRows (H12,H11,H10...)': 'TagLayout::InInverseRows',
+    'InColumnsThenRows (A1,A2,B1,B2...)': 'TagLayout::InColumnsThenRows',
+    'InColumnsThenColumns (A1,B1,A2,B2...)': 'TagLayout::InColumnsThenColumns'
```

Comment:
In columns then columns seems a bit confusing

Reply:
I like the Z but the inverted N seems confusing. What about: InColumnsByQuadInColumns and InColumnsByQuadInRows?